src/test/java/net/sf/jabref/model/entry/BibtexEntryTests.java

```diff
@@ -84,6 +86,22 @@ public void isEmptyCiteKey() {
     }
 
     @Test
+    public void typeOfBibEntryIsMiscAfterSettingToNullString() {
```

Comment:
I would prefer if the entry is setup in the SetUp method using `setField` methods and not parsed from a string.